docs/reference/errors-and-warnings/NU1105.md

```diff
@@ -12,10 +12,29 @@ f1_keywords:
 
 # NuGet Error NU1105
 
-<pre>Unable to read project information for 'ProjectFile'. The project file may be invalid or missing targets required for restore.</pre>
+<pre>Unable to read project information for 'ProjectFile'. The project file may be invalid/missing targets required for restore or the project might be unloaded.</pre>
 
 ### Issue
 The project file exists but no restore information was provided for it.
 
 ### Solution
-In Visual Studio, the error could mean that the project is unloaded, in which case reload the project. From the command line this could mean that the file is corrupt or that it doesn't contain the custom "after imports" target needed for restore to read the project. Check that the project file is valid and contains an "after imports" target.
\ No newline at end of file
+In Visual Studio, the error could mean that the project is unloaded, and thus NuGet was unable to load project information required to restore it. 
+
+#### Solution in VS 2019 16.0 or later:
+- Using a recent CLI version, restore the solution that includes that project from a command line prompt.

```

Comment:
I'm not sure that many customers will understand what "Using a recent CLI version" means. I think it's better to explicitly say "nuget.exe or dotnet cli", or make this phrase a link to other docs that explains what CLI options there are for NuGet. I believe Mike created a page when he was helping us a few months ago.